libs/core/src/components/pds-input/pds-input.tsx

```diff
@@ -137,6 +152,89 @@ export class PdsInput {
    */
   @State() hasFocus = false;
 
+  private updateAddonWidths() {
+    requestAnimationFrame(() => {
```

Comment:
Do we need this? Is this a request from design?

Reply:
Yes, the input value needs to account for the size of the prefix of suffix and apply padding to not have an overflow of text situation

![Screenshot 2025-04-14 at 11 39 51 AM](https://github.com/user-attachments/assets/a136c951-340f-4b39-9fbc-99b73eedfb58)

Reply:
Sorry, when I said "this", I was referring to `requestAnimationFrame`

Reply:
Gotcha. There was an issue where the component would load, but the width would immediately be 0. I used that method instead of adding a setTimeout. Is there a preferred way to accomplish this outside of my proposed solution?